test/unit/formatting_stack/util/diff.clj

```diff
@@ -0,0 +1,47 @@
+(ns unit.formatting-stack.util.diff
+  (:require
+   [clojure.test :refer [are deftest is testing]]
+   [formatting-stack.util.diff :as sut]))
+
+(deftest diff->line-numbers
+  (are [description filename expected] (testing description
+                                         (is (= expected
+                                                (sut/diff->line-numbers (slurp filename))))
+                                         true)
+
+    "only additions yields empty result"
```

Comment:
Can you make all these `description`s more expressive?

Honestly I have trouble understanding them. I'd suggest using phrasing that describes the use cases from a high level perspective. This emphasizes the notion of tests-as-documentation.

(As a rule of thumb: descs should have a verb)